src/Tizen.NUI.Scene3D/src/public/Controls/SceneView.cs

```diff
@@ -287,7 +287,7 @@ public void SelectCamera(string name)
 
         /// <summary>
         /// Starts camera transition from currently selected camera to a camera of index.
-        /// Camera Position and Orientation is smoothly animated.
+        /// Camera Position and Orientation and FieldOfView is smoothly animated.
```

Comment:
Camera Position, Orientation and FieldOfView are smoothly animated.

Reply:
Thank you. fix done.